modules/features/account/src/main/java/au/com/shiftyjelly/pocketcasts/account/onboarding/upgrade/OnboardingUpgradeHelper.kt

```diff
@@ -239,23 +267,100 @@ object OnboardingUpgradeHelper {
     val backgroundColor = Color(0xFF121212)
 
     @Composable
-    fun PlusBackground(
+    fun OldPlusBackground(
+        modifier: Modifier = Modifier,
+        content: @Composable () -> Unit,
+    ) {
+        Box(modifier) {
+            if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.S) {
+                OldBlurredCanvasBackground()
+            } else {
+                ImageBackground(R.drawable.upgrade_background_glows)
+            }
+            content()
+        }
+    }
+
+    @Composable
+    fun UpgradeBackground(
```

Comment:
New background doesn't match Figma specs yet due to complex vector glows. Please ignore reviewing it for now.